shell/platform/android/io/flutter/embedding/engine/dynamicfeatures/PlayStoreDynamicFeatureManager.java

```diff
@@ -293,10 +294,10 @@ public void loadDartLibrary(int loadingUnitId, String moduleName) {
     // performant and robust.
 
     // Search directly in APKs first
-    List<String> apkPaths = new ArrayList();
+    List<String> apkPaths = new ArrayList<String>();
```

Comment:
Android supports Java 7 now. You should be able to just do https://www.baeldung.com/java-diamond-operator

e.g. `new ArrayList<>();`

Reply:
Yep, though I'll clean this up later, let's get the roll through